src/widgets/symbol_item.rs

```diff
@@ -19,12 +20,12 @@ mod imp {
         pub(super) id: RefCell<String>,
         #[property(construct_only, get)]
         pub(super) icon: RefCell<String>,
-        #[property(construct_only, get)]
-        pub(super) package: RefCell<String>,
-        #[property(construct_only, get)]
-        pub(super) command: RefCell<String>,
-        #[property(construct_only, get)]
-        pub(super) mode: RefCell<String>,
+        #[property(construct_only, get, nullable)]
+        pub(super) package: RefCell<Option<String>>,
+        #[property(construct_only, get, nullable)]
+        pub(super) command: RefCell<Option<String>>,
```

Comment:
Considering that `command` is the essential data piece for this widget, I think it's better to keep it as non-nullable and instead filter out the symbols with no Typst command when creating the list.